DatadogSessionReplay/Tests/Feature/RequestBuilder/RequestBuilderTests.swift

```diff
@@ -42,12 +42,12 @@ class RequestBuilderTests: XCTestCase {
         }
 
         // Then
-        XCTAssertEqual(try url(for: .us1), "https://session-replay.browser-intake-datadoghq.com/api/v2/replay")
-        XCTAssertEqual(try url(for: .us3), "https://session-replay.browser-intake-us3-datadoghq.com/api/v2/replay")
-        XCTAssertEqual(try url(for: .us5), "https://session-replay.browser-intake-us5-datadoghq.com/api/v2/replay")
-        XCTAssertEqual(try url(for: .eu1), "https://session-replay.browser-intake-datadoghq.eu/api/v2/replay")
-        XCTAssertEqual(try url(for: .ap1), "https://session-replay.browser-intake-datadoghq.eu/api/v2/replay")
-        XCTAssertEqual(try url(for: .us1_fed), "https://session-replay.browser-intake-ddog-gov.com/api/v2/replay")
+        XCTAssertEqual(try url(for: .us1), "https://browser-intake-datadoghq.com/api/v2/replay")
```

Comment:
Why do we drop session replay subdomain?

Reply:
Since we created first iteration of SR (using above URLs), there was a global change to the way we reference intakes, see: [RUMM-2865](https://datadoghq.atlassian.net/browse/RUMM-2865). Here we migrate SR to the subdomain-less URLs. This is for uniformity with other products in the SDK 👌.